grpc-ballerina/tests/28_unary_basic_auth_client.bal

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/test;
```

Comment:
```suggestion

import ballerina/test;
```

Reply:
It is a result of Ballerina formatter. Even I add explicitly add this space, the formatter will remove that.

Reply:
Ah okay. Then that's a different problem. I am not sure why the formatter does that. So, this is fine.